src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+  
+  return days > 2 && days < 7
+    ? total - 20
+    : days > 6
+      ? total - 50
+      : total;
```

Comment:
Replace magic numbers with named constants to clarify what these numbers represent. For example, use a constant for the `2` and `7` that define the thresholds for discounts, and constants for the discount amounts `20` and `50`. This will make the code more readable and maintainable (checklist item 2).